docs/addons.md

```diff
@@ -46,7 +46,7 @@ spec:
     memoryRequest: "300Mi"
 ```
 
-Read more about cluster autoscaler in the [official documentation](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler).
+Read more about cluster autoscaler in the [official documentation](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler). The autoscaler supports five different expander strategies; the `priority` expander requires additional configuration in a ConfigMap as described in [its documentation](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md).
```

Comment:
I think it is worth adding something like "please make sure that you have deployed this ConfigMap before enabling the priority expander"

Reply:
Sure, I can reword this. FWIW, the cluster-autoscaler docs make it sound like it will fall back to selecting a random node group if the expander selected can't pick one, so it may not actually break things too much if you try to use the `price` expander on AWS or the `priority` expander without a ConfigMap, but I haven't tested that behavior.

Reply:
Added this language to the docs.